core/commands.go

```diff
@@ -0,0 +1,297 @@
+package core
+
+type DiceCmdMeta struct {
+	Name string
+	Info string
+	Docs string
+	Eval func([]string) []byte
+}
+
+var (
+	diceCmds = map[string]DiceCmdMeta{}
+
+	pingCmdMeta = DiceCmdMeta{
+		Name: "PING",
+		Info: "ping command info here",
+		Docs: "ping command docs here",
+		Eval: evalPING,
+	}
+	setCmdMeta = DiceCmdMeta{
+		Name: "SET",
+		Info: "set command info here",
+		Docs: "set command docs here",
+		Eval: evalSET,
+	}
+	getCmdMeta = DiceCmdMeta{
+		Name: "GET",
+		Info: "get command info here",
+		Docs: "get command docs here",
```

Comment:
Assuming Docs is a link to documentation - 
We don't have a docs portal yet. Should we skip this atm? Any placeholders that we won't address in short term is smell for me.
Its easy enough to add it later in one go for all once we do something for docs.

Reply:
Skipping them atm does makes sense. Should I go with empty string or remove the `Docs` field altogther ?
I realize I have a habit of leaving placeholders in my code, and I need to get better at addressing them before submitting my PRs :/

Reply:
I prefer removing whole rather than an empty string if its not doing anything atm.
Let me ask you first, how are you planning to use the docs attribute in future?

Reply:
removed `Docs`.